packages/sdk/src/policies/policies/minMaxInvestmentPolicy.ts

```diff
@@ -23,3 +23,15 @@ export function encodeMinMaxInvestmentPolicySettings({
   }
   return encodeAbiParameters(minMaxInvestmentPolicySettingsEncoding, [minInvestmentAmount, maxInvestmentAmount]);
 }
+
+export function decodeMinMaxInvestmentPolicySettings(settings: Hex) {
+  const [minInvestmentAmount, maxInvestmentAmount] = decodeAbiParameters(
+    minMaxInvestmentPolicySettingsEncoding,
+    settings,
+  );
+
+  return {
+    minInvestmentAmount,
+    maxInvestmentAmount,
+  };
```

Comment:
Create a typescript interface for these and explicitly set the return type of this function with that interface.

E.g. ;see here: https://github.com/enzymefinance/enzyme-sdk/blob/main/packages/sdk/src/fees/fees/performanceFee.ts#L18-L39

And less complicated here: https://github.com/enzymefinance/enzyme-sdk/blob/main/packages/sdk/src/fees/fees/entranceFee.ts#L41-L62

Reply:
We should do that for all.

And then, add those to the exports (see `src/index.ts`).

Whenever you add any functions or types that may be useful for external usage, export it from `src/index.ts`